kits/mou-drafter/apps/public/swiftlatex/index.html

```
@@ -0,0 +1,210 @@
+            <option value="xetex_basic.html">XeTeX basic demo (acmart)</option>
+            <option value="xetex_cjk.html">XeTeX Chinese/Japanese demo</option>
+            <option value="xetex_font.html">XeTeX TrueType demo</option>
+            <option value="xetex_tikz.html">XeTeX Tikz demo</option>
+            <option value="xetex_beamer.html">XeTeX Beamer demo</option>
+            <option value="pdftex_beamer.html">PdfTeX Beamer demo</option>
+            <option value="pdftex_utf8.html">PdfTeX UTF8 demo</option>
+            <option value="pdftex_tikz.html">PdfTeX Tikz demo</option>
+            <option value="pdftex_cv.html">PdfTeX CV demo</option>
+        </select></h6>
+        <h6>It may take a few minutes to download template files for the first time. Please be patient</h6>
+        <iframe src="pdftex_basic.html" id="demoframe" scrolling="no"></iframe>
```

HTMLHint (1.9.2)

[warning] A <iframe> element must have an accessible name.

(frame-title-require)